Framework/PythonInterface/mantid/simpleapi.py

```diff
@@ -166,6 +166,15 @@ def _create_generic_signature_legacy(algm_object):
     return "\b%s" % arg_str, "\b\bVersion=%d" % algm_object.version()
 
 
+def _show_dialog_fn_deprecation_warning(name):
+    """Raise a deprecation warning for a *Dialog being used.
+    """
+    import warnings
+    warnings.warn("{} has been deprecated. "
+                  "All Dialog functions will be removed in a future release.".format(name),
+                  UserWarning)
```

Comment:
The warning is successfully displayed in the script editor output. Just a question regarding readability - would it be better to have it appear in the log or in another colour, say orange, so that the warning is more visible?

Reply:
@giovannidisiena It's a good point and maybe we should also include a logger.warning that would be displayed in the main message display in orange. I'll do that.